.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -14,31 +14,16 @@ jobs:
   timeoutInMinutes: 360
   variables:
     CONDA_BLD_PATH: D:\\bld\\
+    MINIFORGE_HOME: C:\Miniforge
```

Comment:
You should set this in `conda-forge.yml` as well to make this permanant. See conda-smithy docs on how to do that.